lms/djangoapps/course_home_api/dates/v1/serializers.py

```diff
@@ -14,6 +14,7 @@ class DateSummarySerializer(serializers.Serializer):
     """
     Serializer for Date Summary Objects.
     """
+    complete = serializers.NullBooleanField()
```

Comment:
Why allow null?

Reply:
Because not all Date Summary objects have this field defined (read everything but Assignment Dates). So perhaps more of a question of what the frontend would prefer. Should a Course Start date have a completed of False or True or Null?

Reply:
Eh, the frontend can deal with it fine, it just checks falsy.